packages/server/validators/users.js

```diff
@@ -50,7 +51,85 @@ const userIdBodyValidator = [
     }),
 ];
 
+const updateNewUserValidator = [
+  // First Name Check: exist, be between 1 and 50 chars
+  body("firstName", "Invalid firstName.")
+    .exists({ checkFalsy: true })
+    .withMessage("firstName can't be null or empty.")
+    .blacklist("<>")
+    .trim()
+    .isLength({ min: 1, max: 50 })
+    .withMessage("firstName length must be between 1 to 255 characters.")
+    .isAlpha()
+    .withMessage("firstName should be Alphabetic characters."),
+
+  // Last Name Check: exist, be between 1 and 50 chars
+  body("lastName", "Invalid lastName.")
+    .exists({ checkFalsy: true })
+    .withMessage("firstName can't be null or empty.")
+    .blacklist("<>")
+    .trim()
+    .isLength({ min: 1, max: 50 })
+    .withMessage("lastName length must be between 1 to 50 characters."),
+
+  // Pronoun Check: exist, be between 1 and 25 chars, and inside the allowed array.
+  body("pronouns", "Invalid pronoun.")
+    .exists({ checkFalsy: true })
+    .withMessage("pronoun can't be null or empty.")
+    .blacklist("<>")
+    .trim()
+    .isLength({ min: 1, max: 25 })
+    .withMessage("pronoun length must be between 1 to 25 characters.")
+    .isIn(ALLOWED_PRONOUN)
+    .withMessage(
+      "pronouns must be one of the allowed values: " +
+        ALLOWED_PRONOUN.join(", "),
+    ),
+
+  // Major Check: exist, be between 1 and 50 chars
+  body("major", "Invalid major.")
+    .exists({ checkFalsy: true })
+    .withMessage("major can't be null or empty.")
+    .blacklist("<>")
+    .trim()
+    .isLength({ min: 1, max: 25 })
+    .withMessage("Major length must be between 1 to 25 characters."),
+
+  // Avatar Check: exist, be between 1 and 255 chars, and is URL!
+  body("avatar", "Invalid avatar.")
+    .exists({ checkFalsy: true })
+    .withMessage("avatar can't be null or empty.")
+    .blacklist("<>")
+    .trim()
+    .isLength({ min: 1, max: 255 })
+    .withMessage("Name length must be between 1 to 255 characters.")
+    .isURL()
+    .withMessage("Must be an URL"),
```

Comment:
For the avatar URL, it has to be specifically a URL to an image so add a `.matches()` check after this to check against a regular expression to make sure it ends in:
- `.jpg`
- `.png`